2brick/2brick.go

```diff
@@ -0,0 +1,40 @@
+package main
```

Comment:
This is a great solution! It's a little bit dirty, but we have been looking for something like this.

Suggestions:
1. Would it help to sort `a` and `b` as well?
2. The sorting could be done nicer.